src/System.Runtime.InteropServices/src/System/Runtime/InteropServices/ComEventInterfaceAttribute.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+
+namespace System.Runtime.InteropServices
+{
+    [ObsoleteAttribute("ComEventInterfaceAttribute may be unavailable in future releases.")]
+    [EditorBrowsableAttribute(EditorBrowsableState.Never)]
+    [AttributeUsage(AttributeTargets.Interface, Inherited = false)]
+    public sealed class ComEventInterfaceAttribute : Attribute
```

Comment:
Nit: Maybe nice to put it next to similar attributes in Attributes.cs in this directory to follow the style.